examples/metatype.yaml

```diff
@@ -70,6 +70,9 @@ typegates:
         BASIC_andim: hunter2
       func-gql:
         POSTGRES: "postgresql://postgres:password@localhost:5432/db?schema=func_gql"
+      temporal:
```

Comment:
What about `prd`?

Reply:
the `temporal` typegraph is excluded so I don't think it's necessary, I only added this for testing purposes when manually deploying the `temporal` typegraph.